webpack/get_lessons.js

```diff
@@ -0,0 +1,53 @@
+"use strict"
+
+const fs = require('fs')
+const path = require('path')
+
+const LESSON_DIR = path.normalize(path.join(__dirname, '..', 'lessons'))
+
+/**
+ * Retrieves list of lessons.
+ *
+ * @return {Array} Sorted list of lessons.
+ */
+function get_lessons() {
+    function format_dirs(dir_list) {
+        const result = []
+
+        for (let dir of dir_list) {
+            const full_path = path.join(LESSON_DIR, dir)
+
+            if (!fs.lstatSync(full_path).isDirectory()) {
+                continue
+            }
+
+            const name = path.basename(dir)
+
+            if (name === 'default') {
+                continue
+            }
+
+            //format: num-chapter_part1-chapter_part2
+            let [num, ...chapter] = name.split('-')
+            num = parseFloat(num)
```

Comment:
Can we be sure `parseFloat` of `1.3` returns something which will be over 1.3, for formatting purposes (see also my first comment)?

Reply:
My node.js and browser parses it as 1.3, but I can also save string representation if you want to be sure.